Azure/azure-cli
# [Sql] `az sql mi start/stop/start-stop-schedule`: Add SQL MI manual and scheduled start stop

**Related command**
In this PR I add new commands for manual and scheduled start stop for Azure SQL Managed Instance. The commands were generated using the CodeGen tool, while tests were manually written.

**Description**

Here are the new commands with examples:

### Manual start stop:

**Start a managed instance.**
        az sql mi start --ids resourceId
        az sql mi start --mi miName -g resourceGroup --subscription subscription

**Stop a managed instance.**
        az sql mi stop --ids resourceId
        az sql mi stop --mi miName -g resourceGroup --subscription subscription

### Scheduled start stop:

**Create schedule for managed instance.**
        az sql mi startstopschedule create --mi miName -g resourceGroup --subscription subscriptionId --schedule-list "[{'startDay':'Monday','startTime':'10:00 AM','stopDay':'Monday','stopTime':'12:00 AM'}]"

**Delete a managed instance start stop schedule.**
        az sql mi startstopschedule delete --ids resourceId
        az sql mi startstopschedule delete --mi miName -g resourceGroup --subscription subscriptionId

**Lists the managed instance's start stop schedule.**
        az sql mi startstopschedule list --ids resourceId
        az sql mi startstopschedule list --mi miName -g resourceGroup --subscription subscriptionId

**Get the managed instance's start stop schedule.**
        az sql mi startstopschedule show --ids resourceId
        az sql mi startstopschedule show --mi miName -g resourceGroup --subscription subscriptionId


**Testing Guide**
Commands can be tested as shown above in the example.


---

This checklist is used to make sure that common guidelines for a pull request are followed.
